sdk/main/include/ED25519PublicKey.h

```diff
@@ -12,13 +12,16 @@ namespace Hedera
 class ED25519PublicKey : public PublicKey
 {
 public:
-  explicit ED25519PublicKey(unsigned char* encodedPublicKey, size_t keyLength);
+  explicit ED25519PublicKey(const std::vector<unsigned char>& rawPublicKey);
 
   ~ED25519PublicKey();
 
   [[nodiscard]] proto::Key* toProtobuf() const override;
   [[nodiscard]] std::string toString() const override;
 
+  [[nodiscard]] bool verifySignature(const std::vector<unsigned char>& signatureBytes,
+                                     const std::vector<unsigned char>& signedBytes) const override;
```

Comment:
Just thought of it now, but we'll probably want to add documentation comments to this APIs at some point

Reply:
Definitely on my TODOs. I've been holding off on the documentation so far, since I haven't been sure if the stuff I implement will stick around, since OpenSSL is tricky